src/Std/Data/DHashMap/Internal/List/Associative.lean

```diff
@@ -26,6 +27,8 @@ open List (Perm)
 
 namespace Std.DHashMap.Internal.List
 
+attribute [-simp] List.isEmpty_eq_false
+
```

Comment:
@TwoFX, I'd like to have this lemma on globally, but it interferes with the way you've set things up here. I suspect your preference will be to turn it off locally, so you can keep your lemma names parallel with other files. Let me know if we should add a comment. (Feel free to push anything.)